pkg/registry/storage/csidriver/strategy_test.go

```diff
@@ -222,116 +244,123 @@ func TestCSIDriverPrepareForUpdate(t *testing.T) {
 			StorageCapacity: &disabled,
 		},
 	}
+	driverWithServiceAccountTokenGCP := &storage.CSIDriver{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+		},
+		Spec: storage.CSIDriverSpec{
+			ServiceAccountTokens: []storage.ServiceAccountToken{{Audience: &gcp}},
+			RequiresRemount:      &enabled,
+		},
+	}
+	driverWithServiceAccountTokenVault := &storage.CSIDriver{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "foo",
+		},
+		Spec: storage.CSIDriverSpec{
+			ServiceAccountTokens: []storage.ServiceAccountToken{{Audience: &vault}},
+			RequiresRemount:      &enabled,
+		},
+	}
 
-	var resultEmpty []storage.VolumeLifecycleMode
+	// var resultEmpty []storage.VolumeLifecycleMode
```

Comment:
how come we got rid of these?

Reply:
oops. i accidentally commented them out.